subsys/testsuite/ztest/src/ztest.c

```diff
@@ -1321,15 +1326,57 @@ static int cmd_shuffle(const struct shell *sh, size_t argc, char **argv)
 
 static int cmd_run_suite(const struct shell *sh, size_t argc, char **argv)
 {
+	struct getopt_state *state;
+	int opt;
+	static struct option long_options[] = {{"repeat_iter", required_argument, 0, 'r'},
+		{0, 0, 0, 0}};
```

Comment:
Could change the second element to `{NULL, 0, NULL, 0}` as they are pointers. `0` and `NULL` of course are the same, but some people prefer to always use `NULL`s for pointers.

Reply:
Done.